src/cmd/create.go

```diff
@@ -710,18 +710,42 @@ func pullImage(image, release, authFile string) (bool, error) {
 		shouldPullImage = true
 	}
 
+	imageSizeChan := make(chan string)
+
 	if promptForDownload {
 		fmt.Println("Image required to create toolbox container.")
+		// Launch a goroutine to call getImageSizeFromRegistry and send the size to the channel
+		go func() {
+			imageSize, err := getImageSizeFromRegistry(imageFull)
+			if err != nil {
+				logrus.Debugf("Getting image size failed: %s", err)
+				imageSize = ""
+			}
+			imageSizeChan <- imageSize
+		}()
+
+		// Show the download question immediately
+		prompt := fmt.Sprintf("Download %s (...)? [y/N]:", imageFull)
+		fmt.Printf("%s", prompt)
```

Comment:
The prompt is being shown immediately, but the user can't interact with it.  It should use `askForConfirmation()` and run in a separate goroutine.

Reply:
I think I've solved this.